features/step_definitions/metering.rb

```diff
@@ -225,3 +224,47 @@
   res = host.exec(shell_cmd)
   raise "#{cb.metering_namespace} #{op} unsuccessfully" unless res[:success]
 end
+
+# install metering via OLM,
+Given /^the#{OPT_QUOTED} metering service is installed(?: to $QUOTED)? using OLM$/ do | metering_ns |
+  ensure_admin_tagged
+  # 1. create the metering namespace
+  metering_ns ||= "openshift-metering"
+  project(metering_ns)
+  step %Q/I switch to cluster admin pseudo user/
+  @result = user.cli_exec(:create_namespace, name: metering_ns)
+  # prep for OLM need to define configs via YAML
+  # 1. metering-catalogsourceconfig.yaml
+  catalog_source_config = "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/metering/OLM_examples/metering-catalogsourceconfig.yaml"
```

Comment:
Hi @pruan-rht  Why don't we use test files from `https://github.com/operator-framework/operator-metering/tree/master/manifests`?

Reply:
@yapei because the OLM install is still very experimental, so I don't want to put those config into the official metering github repo.

Reply:
It is preferable to have things pre-installed on cluster install instead of whenever a test runs. If we keep it like this, then making step `ensure_destructive_tagged` is needed to avoid installing this at the same time by two scenarios.

Another issue is in case you want at some point to insall with a different configuration. How would you handle it?